plutus-ledger-api/src/Plutus/V1/Ledger/Slot.hs

```diff
@@ -42,7 +42,8 @@ import           Plutus.V1.Ledger.Interval
 newtype Slot = Slot { getSlot :: Integer }
     deriving stock (Haskell.Eq, Haskell.Ord, Haskell.Show, Generic)
     deriving anyclass (FromJSON, FromJSONKey, ToJSON, ToJSONKey, NFData)
-    deriving newtype (Haskell.Num, AdditiveSemigroup, AdditiveMonoid, AdditiveGroup, Haskell.Enum, Eq, Ord, Haskell.Real, Haskell.Integral, Serialise, Hashable, PlutusTx.IsData)
+    deriving newtype (AdditiveSemigroup, AdditiveMonoid, AdditiveGroup, Eq, Ord, Enum, PlutusTx.IsData)
+    deriving newtype (Haskell.Num, Haskell.Enum, Haskell.Real, Haskell.Integral, Serialise, Hashable)
```

Comment:
Curious, why not combine the two `deriving newtype`?

Reply:
I decided to split into two groups, Haskell-related and Plutus-related, because the line is too long. If it makes no sense or strange I can get it back.

Reply:
Good idea